cbg_executor_demo/src/cbg_executor_demo/pong_node.cpp

```diff
@@ -0,0 +1,105 @@
+// Copyright (c) 2020 Robert Bosch GmbH
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "cbg_executor_demo/pong_node.hpp"
+
+#include <cassert>
+
+#include <chrono>
+#include <memory>
+
+#include "cbg_executor_demo/parameter_helper.hpp"
+
+namespace cbg_executor_demo
+{
+
+PongNode::PongNode()
+: rclcpp::Node("pong_node")
+{
+  using std::placeholders::_1;
+  using std_msgs::msg::Int32;
+
+  declare_parameter<double>("high_busyloop", 0.01);
+  high_pong_publisher_ = create_publisher<Int32>("high_pong", rclcpp::SensorDataQoS());
```

Comment:
other demos use the `this` pointer explicitly.
```suggestion
  high_pong_publisher_ = this->create_publisher<Int32>("high_pong", rclcpp::SensorDataQoS());
```

same below.